helm/defectdojo/templates/secret.yaml

```
@@ -16,6 +16,12 @@ metadata:
     helm.sh/resource-policy: keep
     helm.sh/hook: "pre-install"
     helm.sh/hook-delete-policy: "before-hook-creation"
+    {{- with .Values.secretAnnotations }}
```

```suggestion
    {{- with .Values.secrets.annotations }}
```
See note in `values.yaml`